src/main/java/frc/robot/subsystems/Intake.java

```diff
@@ -10,32 +11,36 @@
 
     public static final WPI_VictorSPX intakeMotor = new WPI_VictorSPX(IntakeConstants.INTAKE_WHEELS_ID);
     public static final WPI_VictorSPX intakeBarMotor = new WPI_VictorSPX(IntakeConstants.INTAKE_BAR_MOTOR_ID);
+    public DoubleSolenoid test = new DoubleSoleniod(IntakeConstants.PCM_ID, IntakeConstants.FORWARD_CHANNEL, IntakeConstants.REVERSE_CHANNEL);
+    public IntakeMotor = new IntakeMotor(IntakeConstants.FIRST_MOTOR_CONTROLLER_ID, IntakeMotor.SECOND_MOTOR_CONTROLLER_ID, IntakeMotor.INWARD, IntakeMotor.OUTWARD, IntakeMotor.STOP);
+    public DigitalInput input = new DigitalInput(0);
+
 
     public Intake(){
     }
 
     public void extend() {
-        intakeBarMotor.set(ControlMode.PercentOutput, IntakeConstants.INTAKE_BAR_MOTOR_SPEED);
+        test.set(DoubleSolenoid.Value.kforward);
     }
 
     public void retract() {
-        intakeBarMotor.set(ControlMode.PercentOutput, -(IntakeConstants.INTAKE_BAR_MOTOR_SPEED));
+        test.set(DoubleSolenoid.Value.kreverse);
     }
 
     public void getPosition() {
-        
+        input.get(DigitalInput);
     }
 
     public void spinIn(){
-        intakeMotor.set(ControlMode.PercentOutput, IntakeConstants.INTAKE_FLYWHEELS_FORWARD_POWER);
+        test.set(IntakeMotor.Value.inward);
```

Comment:
you set motor speed like, motor_name.set(ControlMode.PercentOutput, NAME OF YOUR CONSTANT)